### ISSUE TITLE
TypeError: "t" is undefined

### ISSUE THREAD

**Issue by SoCuul** [photoview/photoview#903]
**Describe the bug**
When I scroll beyond a few thousand photos, clicking on a photo renders this error and forces me to reload the page to continue using the site.

**Screenshots**
![image](https://github.com/photoview/photoview/assets/63339559/d419583f-93b0-45a9-8470-b88d503c2d72)

**Your environment:**
 - Are you using Docker? **yes**
 - Database: mariadb:10.5
 - Browser: Firefox Stable 119.0 (macOS)
 - Photoview version [e.g. 2.0.0]: 2.3.13

**Comment by SoCuul** [comment]
@kkovaletp

**Comment by SoCuul** [comment]
@viktorstrate you come into contact with this before?

**Comment by SoCuul** [comment]
@kkovaletp Do you have this problem while using your instance?

**Comment by kkovaletp** [comment]
@SoCuul, no I don't.
I have 6.5 K media files in my gallery and scrolled them all in my latest stable Chrome with opened DevTools: no single error has been reported, even though I have a few corrupted photos, so no thumbnails were displayed for them

**Comment by SoCuul** [comment]
> @SoCuul, no I don't. I have 6.5 K media files in my gallery and scrolled them all in my latest stable Chrome with opened DevTools: no single error has been reported, even though I have a few corrupted photos, so no thumbnails were displayed for them

That's super strange. Could you possibly try scrolling through them in Firefox? That could narrow down if the issue is my personal instance.

**Comment by kkovaletp** [comment]
Please note that my instance is built from #910 code with many fixes applied compared to the `master`

**Comment by kkovaletp** [comment]
> Could you possibly try scrolling through them in Firefox?

the result is the same: no errors

**Comment by SoCuul** [comment]
> From my testing on windows, this problem only occurs in firefox (
> 
> 125.0.2) , but everything seems to be fine in chrome (124.0.6367.60)
> 
> 
> 
> Maybe it has something to do with the plug-ins enabled in firefox? over 10000+ photo

I could give you the list of extensions I have installed to help troubleshoot it? If you'd be willing to try that.

**Comment by SoCuul** [comment]
Also, is there any docker image built from the master? Maybe that could fix it.

**Comment by kkovaletp** [comment]
> Also, is there any docker image built from the master? Maybe that could fix it.

yes, there is the `photoview:master` image in the Docker Hub registry: https://hub.docker.com/r/viktorstrate/photoview/tags
You need to change the next line in your docker-compose.yml for the photoview service:
`image: viktorstrate/photoview:master`

**Comment by kkovaletp** [comment]
@SoCuul, any updates about this issue? Is it fixed in the latest `master` image?

**Comment by SoCuul** [comment]
> @SoCuul, any updates about this issue? Is it fixed in the latest `master` image?

I will check when I have time
